packages/renderer/src/App.svelte

```diff
@@ -178,6 +178,12 @@ window.events?.receive('kubernetes-navigation', (args: unknown) => {
         <Route path="/images/existing-image-create-container" breadcrumb="Select image" >
           <CreateContainerFromExistingImage />
         </Route>
+        <Route path="/images/build" breadcrumb="Build an Image">
+          <BuildImageFromContainerfile />
+        </Route>
+        <Route path="/operations/images/build/:taskId" breadcrumb="Build an Image" let:meta>
```

Comment:
remark (blocking): I am not sure to understand why need two routes ? we could use instead a query parameter, so we have a single root, and have the taskId `string | undefined`.

Reply:
resolved